src/compiler/diagnosticMessages.json

```diff
@@ -2466,5 +2466,9 @@
     "A constructor cannot contain a 'super' call when its class extends 'null'": {
       "category": "Error",
       "code": 17005
+    },
+    "Left hand side of '**' cannot be a simple unary expression. Consider parenthesize the expression.": {

```

Comment:
Ron's error message is good too. I'd just rather you called it an exponentiation expression instead of a '**' expression.